cypress/e2e/DateCheck.feature

```diff
@@ -17,32 +19,27 @@ Feature: Check signature dates
     And I can see that the donor "Mr Dead Pool" can sign
 
   Scenario: Displays relevant information on signing continuation sheets
-    Given I visit the dashboard
-    And I click "check-signing-dates" for LPA ID 26997335999
-    Then I am taken to "/lpa/26997335999/date-check"
+    Given I visit "/lpa/26997335999/date-check"
     # Additional attorneys (cs1), additional preferences (2), donor cannot sign or make mark (3)
-    And I can see a reminder to sign continuation sheet 1, 2 and 3
+    Then I can see a reminder to sign continuation sheet 1, 2 and 3
     # primaryAttorney is a trust corporation (4)
     And I can see a reminder to sign continuation sheet 4
     And I can see that the donor "Mr Christopher Robin" cannot sign
 
   Scenario: Displays the correct references to donor/applicant in errors when they cannot sign
-    Given I visit the dashboard
-    When I click "check-signing-dates" for LPA ID 26997335999
-    And I click element marked "Check dates"
-    And the visually-hidden legend for "date-check-donor-date" states "Check signature dates for the person who signed on behalf of the donor"
-    Then I can see validation errors refer to the person signing on behalf of the donor, who is also the applicant
+    Given I visit "/lpa/26997335999/date-check"
```

Comment:
The specific url here bothers me somewhat.  The way it was before , didn't specify the exact url, so that could move. This way we are tied to a specific path.  I suppose its unlikely to change, but still.   Also the hard-coded LPA ID bothers me too a bit. But I suppose there has to be some way of identifying the exact test fixture we want to look at

Reply:
The LPA ID has to be there to get the fixture we want, as you say. As for the path: the alternative is to search for the LPA ID, then click on the link. That would remove the hard-coded path, but would introduce about 3 extra steps for each test case (load the dashboard, enter text in the search box, press search, click on the link). We could do that if you think it's important. On the other hand, these paths are unlikely to change; and if they do, the tests will break and we can fix them.

Reply:
OK

Reply:
Would you prefer if the path was hidden behind some plain English?

Reply:
Actually that might be a good thing, yes. :) It just hides the implementation so that in theory if the link moved or something, tests wouldn't need to change.

Reply:
I'll do that.